var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -71,6 +71,9 @@ class Apex(CMakePackage):
     conflicts('+jemalloc', when='+gperftools')
     conflicts('+plugins', when='~activeharmony')
 
+    # Patches
```

Comment:
Can you turn this comment into a brief description of what the patch does and why it is needed?

Reply:
I left the `# Patches` heading to fit the other sections (`# Dependencies`, `# Conflicts`, etc.), but added a comment specifically for this patch.